pkg/network/node/node.go

```diff
@@ -355,6 +355,44 @@ func (node *OsdnNode) validateMTU() error {
 		if resultErr != nil {
 			return fmt.Errorf("could not update node with taint after many retries: %v", resultErr)
 		}
+	} else if tainted && int(node.networkInfo.MTU) <= 50 {
+		resultErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+			klog.V(2).Infof("Node has too small MTU taint but MTU is big enough, untainting node...")
+
+			oldNode, err := json.Marshal(n)
+
+			if err != nil {
+				return fmt.Errorf("could not marshal old Node object: %v", err)
+			}
+
+			nodeClone := n.DeepCopy()
+
+			untaintedNode, _, err := taints.RemoveTaint(nodeClone, &corev1.Taint{Key: MTUTaintKey, Value: "value", Effect: "NoSchedule"})
+
+			if err != nil {
+				return fmt.Errorf("could not untaint the node with key %s: %v", MTUTaintKey, err)
+			}
+
+			newNode, err := json.Marshal(untaintedNode)
+
```

Comment:
As a random style note, golang code (or maybe just kubernetes code?) tends to not use so many blank lines. In particular, you should never put a blank line between setting `err` and checking it.